lib/pagination_search/search_conditions.rb

```diff
@@ -37,8 +37,16 @@ def wrapped(cleaned_grouped_hash)
         Result.new(cleaned_grouped_hash)
       end
 
+      def quotation_safe(search_string)
+        if search_string.count('""').odd? || /\S\"\S/.match(search_string)
```

Comment:
Given that this case you mention now does not return an error, wouldn't that be a different story?